doc/api/fs.md

```diff
@@ -586,11 +586,11 @@ changes:
                  buffers anymore.
 -->
 
-* `buffer` {Buffer|TypedArray|DataView|string|Object}
+* `buffer` {Buffer|TypedArray|DataView}
```

Comment:
We probably need to add an entry to the changes list for this method as well.

Reply:
Yup, rebasing~

Reply:
We'd still want an entry status for this `filehandle.write` I think https://github.com/nodejs/node/blob/ae1f6db0886a540dcf33f27bb4f13c37e0c8899e/doc/api/fs.md#L589-L597

Reply:
Ugh sorry, I've misread the signature.

This one is separated from `fh.write(string, ...)` in docs, so I consider string-related behaviours to be irrelevant here. The mentioned change is just a doc-only correction. I.e. it doesn't matter if first param is `string` or `string|Object`, because these params belong to "another method" anyway.

I think it would make more sense to keep the `v14.12.0 | The string parameter will stringify an object with an explicit toString function.` entry for `fh.write(string, ...)` method, but omit it in `fh.write(buffer, ...)` versions to avoid confusion.
Does it sound better?

Reply:
Yes I think that would work – as long as the history doesn't imply that it's still supported, we should be good.

Reply:
We also want to update these ones I think: 
https://github.com/nodejs/node/blob/f3b05ddf63d29e4f49ba405b25bc52d52279ee2f/doc/api/fs.md#L4442-L4450
https://github.com/nodejs/node/blob/f3b05ddf63d29e4f49ba405b25bc52d52279ee2f/doc/api/fs.md#L4469
https://github.com/nodejs/node/blob/f3b05ddf63d29e4f49ba405b25bc52d52279ee2f/doc/api/fs.md#L4526-L4529
https://github.com/nodejs/node/blob/f3b05ddf63d29e4f49ba405b25bc52d52279ee2f/doc/api/fs.md#L4555
https://github.com/nodejs/node/blob/f3b05ddf63d29e4f49ba405b25bc52d52279ee2f/doc/api/fs.md#L5746-L5749
https://github.com/nodejs/node/blob/f3b05ddf63d29e4f49ba405b25bc52d52279ee2f/doc/api/fs.md#L5767

Reply:
Eventually. Coercion in `fs.write`, `fs.writeFile` and `fs.writeFileSync` (as well as `fs.appendFile`, `fs.appendFileSync`) is doc-only deprecated ([DEP0162](https://github.com/nodejs/node/commit/4b198ffa7e681824dea10ed372a9ed7201c2c7f5)) atm; so these will be updated later.

Reply:
If we want to make that a runtime deprecation for v18.x, we should hurry, the semver cutoff is next week.

Reply:
Affirmative. I can't promise anything right now, but will try to do it in time.